src/components/Quit.js

```diff
@@ -0,0 +1,33 @@
+// returns to the state map
+// reset score
+//remove any choice
+// Display the answer (Town, County, Longitude, and Latitude)
+import { useState } from "react";
+function Quit() {
+  // alert(<Info/>)
+  return (
+    <div>
+      <button id = "Quit" onClick={() => window.location.reload(false)}>Quit</button>
+    </div>
+  );
+  function Alert(props) {
+    const [display, setDisplay] = useState(false);
+    let clickMe = display;
+
+    setDisplay(!clickMe);
+    return (
+      <div style={{ height: "250px", width: "100%", background: "yellow" }}>
+        <h1>Hello!! I am MODAL!</h1>
+        <button
+          onClick={(evt) => {
+            setDisplay(true);
+          }}
+        >
+          Close
+        </button>
+      </div>
+    );
+  }
+}
```

Comment:
I'm not sure what the intentions of this component are but the structure is quite off the return should be the last thing inside of this function and your function Alert should exist up above that. There are also two different returns happening and they are both doing different things (which can be an okay thing in react but this is not set up in that way)